environment.yml

```diff
@@ -4,5 +4,7 @@ channels:
 dependencies:
   - python=3
   - mkdocs
-  - markdown-checklist
   - pymdown-extensions
+  - pip
+  - pip:
+    - markdown-checklist
```

Comment:
This one was update in conda today. You can revert these changes in a future PR. No need to do it now.

Reply:
Will do. Thanks for the heads up!